README.md

```diff
@@ -23,120 +25,192 @@ asdf plugin-add nodejs
+1. Clone this repository locally unless you've already done so:
+git clone git@github.com:the-difference-engine/rescue-leftover-cuisine2.git
 
-```
-cp .env.example .env
-```
+2. in your terminal, navigate locally to the rescue-leftover-cuisine2 folder
 
-This will provide the minimum you need to run the site on your local computer. If you need to work on signup/confirmation/forgot password workflows, you'll also need to set the sendgrid configuration to enable sending emails. Talk to a tech lead if you need those credentials.
+3. run 'bundle install'
 
-## Docker
+4. run 'rails server' or 'rails s' to start the server
 
-### New to Docker?
-If you are completely new to Docker, please see the [Getting Started with Docker Guide](https://docs.docker.com/get-started/) on Docker's official website. Your first exposure to using containers can be confusing, so please don't hesitate to ask your team's Technical Lead for help.
+5. Change your current working directory to the `frontend` folder (`cd frontend` if you're currently in the project root)
 
-### Install Docker
-Select the installation guide below for your laptop's operating system:
-- [**Mac OSX**](https://docs.docker.com/docker-for-mac/install/)
-- [**Linux**](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#set-up-the-repository)
-- [**Windows**](https://docs.docker.com/docker-for-windows/install/)
+6. Run `npm install`
 
-### Install Docker Compose
-#### Mac OSX & Windows
-For **Mac OSX** and **Windows** users, Docker Compose is already included with the Docker toolbox.
+7. Run `PORT=5000 npm start`
 
-#### Linux
-For linux users, you will have to install the docker compose binary from the [Docker Compoose release repository](https://github.com/docker/compose/releases). Please follow the instructions below to properly install docker compose:
+8. Navigate to 'http://localhost:5000' in your browser
 
-1. Run this command in your terminal to install the Docker Compose binary
 
-```
-sudo curl -L https://github.com/docker/compose/releases/download/1.22.0-rc2/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
-chmod +x /usr/local/bin/docker-compose
-```
+## Steps to develop locally
 
-2. Apply [executable permissions](https://ryanstutorials.net/linuxtutorial/permissions.php) to the binary
+1. Ask your product manager to assign you on a Zenhub ticket.
+Ask questions and try to understand your requirement.
 
-```
-sudo chmod +x /usr/local/bin/docker-compose
+2. Make a new git branch off of the staging branch
+
+3. On the new feature branch, start developing your solution. Try to commit often to be able to follow your progress.
+
+4. Feel free to use rails console and http://localhost:3000 to visually test your changes.
+
+5. Push you branch up: 'git push origin *branch_name*' or 'git push origin HEAD'
+
+6. In Github, make a Pull Request (PR) and follow up on the comments you receive, in order to make code changes.
```

Comment:
It may be good to expand on this process a little bit. Something like this:

```
6. In Github, make a Pull Request (PR), and make sure a tech lead or mentor is marked as a reviewer or assignee

  6a. Follow-up on the reviewer's feedback, and implement any requested code changes in your local branch

  6b. Push your local changes up to Github, making sure your local branch is up-to-date with the latest version of the `staging` branch

  6c. Re-request a review from a tech lead or mentor

  6d. Repeat steps 6a through 6c until your PR is approved

  6e. Merge your branch into Github
```

Feel free to trim that down if you think it's overkill.

Reply:
I think having sub-items is a great idea! Thanks so much for the idea!